docker/dev/ubi/build.sh

```diff
@@ -2,11 +2,15 @@
 
 set -x
 
-docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$https_proxy --build-arg NO_PROXY=$no_proxy -t noirolabs/ubibase-opflex-build-base -f Dockerfile-ubibase-opflex-build-base
-docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$https_proxy --build-arg NO_PROXY=$no_proxy -t noirolabs/ubibase-opflex -f Dockerfile-ubibase-opflex
-docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$https_proxy --build-arg NO_PROXY=$no_proxy -t noirolabs/ubibase-aci -f Dockerfile-ubibase-aci
+export http_proxy=http://proxy.esl.cisco.com:80
```

Comment:
I don't think we should explicit refer to internal proxies. It should come from env vars

Reply:
sure, this script is not used by the code but by me to build base images so we can run the UBI builds outside of the build servers without the certificates. I did not want to set the env variables each time i ran the script. so it was a convenience thing.